client/src/App.js

```diff
@@ -303,8 +331,50 @@ export default class NormalDisplay extends React.Component {
                 </pre>
     }
 
+    async checkJournalReady(journalUrl) {
+        if (this.isAborted) return;
+        let resp;
+        try {
+            var data = new FormData();
+            data.append('journalUrl',journalUrl);
+            resp = await fetch("/api/check-jrnl-ready", {
+                method: 'POST',
+                headers: {
+                    Authorization: this.state.devToolToken
+                },
+                body: data
+                });
+            if (!resp.ok) {
+                const errorMessage = await this.formatErrorMessage(resp, '/check-jrnl-ready')
+                throw new Error(errorMessage);
+            }
+            resp = await resp.json();
+            return resp;
+        } catch(e) {
+            console.log(e);
```

Comment:
In other places, like for instance, https://github.com/adobe/asset-compute-devtool/blob/f165b4bda8d00ec7be9865dccc1bfe18679995ad/client/src/App.js#L112, a `Log` function is used instead of `console.log`. Please also use it for consistency (if this is just a debug statement, then remove the `console.log`).